src/ocamlorg_frontend/components/toc.eml

```diff
@@ -29,8 +29,8 @@ let render (t : t) =
             <ol>
               <% items |> List.iter begin fun item -> %>
               <li>
-                <a href="<%s item.href %>" class="text-legacy-default py-1 md:py-0 pl-5 pr-1 block transition-colors border-l-4 border-t-2 border-b-2 border-r-2 border-transparent rounded"
-                  :class='isWithin(scrollY, "<%s item.href %>", sectionYPositions) ? "font-semibold border-l-primary text-primary bg-legacy-primary-200": "hover:text-primary"'
+                <a href="<%s item.href %>" class="text-content dark:text-dark-content py-1 md:py-0 pl-5 pr-1 block transition-colors border-l-4 border-t-2 border-b-2 border-r-2 border-transparent rounded"
+                  :class='isWithin(scrollY, "<%s item.href %>", sectionYPositions) ? "font-semibold border-l-primary dark:border-l-dark-primary text-primary dark:text-dark-primary bg-primary_nav_block_hover_10 dark:bg-primary_nav_block_hover_10": "hover:text-primary dark:hover:text-dark-primary"'
```

Comment:
I'm a bit unclear why the light and dark mode colors are both `bg-primary_nav_block_hover_10` is that according to the figma design?